common/transport/http/src/rest_api_client.ts

```diff
@@ -176,11 +173,7 @@ export class RestApiClient {
     if ((!this._accessToken) || this.isAccessTokenCloseToExpiry(this._accessToken)) {
       this._accessToken = await this._config.tokenCredential.getToken(this._iotHubPublicScope) as any;
```

Comment:
this._iotHubPublicScope is now a sring. does getToken() change?

Reply:
See above comment